### DISCUSSION TITLE
No DNS issues, nginx: host not found in upstream

### DISCUSSION THREAD

**Discussion by awoimbee** [containers/podman-compose#834] [Show and tell]
For poor souls like me with this issue:
1. Install aardvark/netavark
2. Enable DNS:
    docs: https://github.com/containers/podman/blob/main/docs/tutorials/basic_networking.md#bridge
    commands:
    ```bash
    mkdir -p ~/.local/share/containers/storage/networks/
    podman network inspect podman | jq '.[]' > ~/.local/share/containers/storage/networks/podman.json
    # open that json file, set `dns_enabled` to true
    ```

**Comment by muayyad-alsadi** [comment]
I believe if your network is created after installing aardvark/netavark and enabling it
this should be done automatically in `/usr/share/containers/containers.conf` or `/etc/containers/containers.conf ` 

```
[network]
network_backend="netavark"
```

then

```
podman system reset --force
```

you should see something like

```
$ podman info | grep  net
  networkBackend: netavark
```

make sure you do *not* see `network-backend:"cni" but instead have `network-backend:"netavark"`

**Reply by awoimbee** [reply]
Thanks for the response.

Other people with this issue should clearly follow this message first.
I don't know what I missed, but at least it works for me now.

Note that I use openSUSE Tumbleweed and that new podman installs automatically install netavark, it was clearly an issue of upgrading my environment.
